### PR TITLE
Update BSD.md, remove editor install

### PR DESCRIPTION
**Pull request by craigcomstock**
removed annoying opinionated editor install ;) make this easy to use for anyone and not install software that is not completely necessary. A user will have an editor installed already most likely.

### CODE REVIEW

File: docs/BSD.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Tested on FreeBSD 12.1 VM in AWS EC2.
 ## Dependencies
 
 ```
-$ su root -c 'pkg update -f && pkg install -y gdb gcc gmake lmdb autoconf automake libtool git python3 emacs-nox'
+$ su root -c 'pkg update -f && pkg install -y gdb gcc gmake lmdb autoconf automake libtool git python3'
```
Comment (nickanderson) [inline docs/BSD.md:10]: LOL, BOOOOOOO

**Review by nickanderson — APPROVED** [review]

File: docs/BSD.md (hunk 1 above)
Comment (craigcomstock) [reply docs/BSD.md:10]: editor 🔥 merge this PR and the 🔥  is gone ;) and I don't have to edit the line when running it with `.w !sh` in `ed(1)` ;)

Comment (craigcomstock) [reply docs/BSD.md:10]: while we're at it, we should remove the preceding `$ ` and `# ` from stuff like this... I am always having to delete it with `s/^$//` or `s/^#//` before running `.w !sh` :p

**Comment by craigcomstock** [comment]
no need to cherry pick ;)

File: docs/BSD.md (hunk 1 above)
Comment (olehermanse) [reply docs/BSD.md:10]: @craigcomstock IMO this just makes it slightly more annoying. Whenever I tested something on BSD, I didn't have an editor, so I need to install something, and having this command ready helped :)

**Pull request merged**